database/database.sql

```diff
@@ -0,0 +1,94 @@
+/*To create this database locally in your system  follow the following steps
+
+	1. Install PostgreSQL locally in your system . Follow the following steps to intsall it if not installed in ubuntu
+		1.Open the terminal (Ctrl+Alt+T)
+		2. Type $ sudo apt-get update
+		3. Type $ sudo apt-get install postgresql postgresql-contrib 
+	2. Switch over postgres account on your server by typing 
+		$ sudo -i -u postgres
+	3. Now access the postgres prompt by typing 
+		$ psql
+
+	4.Now create the database 'Transport_Mangement_System' by typing the following command
+		CREATE DATABASE Transport_Management_System ;
+
+	5. Type the following command to connect/select this database
+		# \c Transport_Mangement_System
+	
+	6. To Create the relations locallcy in the system run the following queries in your system*/ 
+
+CREATE TABLE Stu_Per_Data (
+	USN VARCHAR PRIMARY KEY NOT NULL,
+	Name CHAR(30) NOT NULL,
+	Email VARCHAR NOT NULL,
+	Mobile_No VARCHAR NOT NULL,
+	Status boolean NOT NULL DEFAULT false,
+	latitude real,
+	longitude real,
+	password VARCHAR 
+);
+
+CREATE TABLE USN_UID(
```

Comment:
Why do you have separate table for mapping usn with uid?

Reply:
In tables like Stu_Trip_Data , Cancel_Trip UID will be used as primary key. So in the table from where it is referenced it should be primary key. 
Now in table Stu_Per_Data , 'UID' cannot be primary key because UID is assigned only after admin approves the student. Till that time, UID for a particular student will be NULL.
So to make UID a primary key in a relation and to get the reference  between USN and UID it is made a separate table.
The relation USN_UID will be filled automatically by the system when admin approves the student(i.e after payment of fees)

Reply:
Got it